src/main/resources/application.yaml

```diff
@@ -62,6 +62,8 @@ serviceRequestClaimIssued:
     site_id: AAA7
     service: CIVIL
 
+wa:
+  feature-toggle: WA3.5
```

Comment:
@nagaraju-deekonda You were using a variable here till yesterday, Why has this changed?